0000-interledger-eli5/0000-interledger-eli5.md

```diff
@@ -0,0 +1,124 @@
+
+That same connector also happens to have an account for JPY at the same bank as the receiver.  So from ILP's point of view _the connector and the receiver have accounts on the same ledger._
+
+If a connector has accounts on the same ledger as the sender and the receiver, an ILP payment involving the connector is now possible.
+
+Let's say the connector offered the sender a price of ¥105 Japanese Yen for each $1 USD, and the sender was happy with that price.  That would mean the sender would send $10 USD, and the receiver would receive ¥1050 JPY (¥105 x $10 = ¥1050).
+
+Here's what happens next:
+
+1) The sender says to the ledger manager of the USD ledger: 
+
+- "I've decided to send $10 USD. Can you write in the ledger book that its not in my account anymore?"
+- "Then, can you write that this $10 is in the ledgers special 'holding' account?" 
+- "Also, here's four special boxes.  Keep one, and give the other three to this connector."
+- "Finally, tell the connector if they give you a password that opens this box, you'll erase the entry in the holding account, and record instead that the $10 USD is in the connectors account!"
+
+2) The ledger manager erases $10 USD from the senders account, puts $10 USD in the holding account, tells the connector about everything, keeps a box, and gives the connector the other two boxes.  Now, its the connectors turn to do a similar thing on the JPY ledger.  The connector says to the JPY ledger manager: 
+
+- "I'm going to give someone on your ledger ¥1050 JPY.  Can you take ¥1050 from my account, and put it in your special 'holding' account?
+- "Then, can you take these boxes, keep one, and give the other to the receiver?"
+- "Finally, wait for a password from the receiver - if the password opens your box, put the ¥1050 in the receivers account, tell the receiver you've done that, and let me know too!"
+
+3) The ledger manager does all this, and gives the receiver the last box.  The receiver hears all about the payment and puts the password they previously arranged with the sender into the box.  The box opens!  So, the receiver knows this box came from the sender, and that this payment is a real payment the sender wants to make.  The receiver also knows that right now, some Japanese Yen are on hold for the receiver in the JPY ledger, waiting for this password.  The receiver wants the money, and trusts the JPY ledger manager, so gives the JPY ledger manager the password.
+
+4) The JPY ledger manager puts the password in the box, and it opens! The ledger manager is now sure that the box came from the original sender, and that the receiver and the sender agreed and are happy with whats happening. The JPY ledger manager changes the entries in the ledger, so that the holding account doesn't have the ¥1050 JPY, and the receiver's account has it instead.  The JPY ledger manager tells the connector what happened, and because the JPY ledger manager trusts the connector, gives the connector the password.
```

Comment:
Also the ledger manager doesn't "trust" the connector, it's just that the connector has an account on that ledger and because money was transferred from them they're entitled to know why